packages/graphql-persisted/src/apollo.ts

```diff
@@ -46,7 +50,7 @@ export class PersistedLink extends ApolloLink {
 
       operation.setContext({
         http: {
-          includeQuery: false,
+          includeQuery: alwaysSendQuery,
```

Comment:
I'd probably change this option to match the context field name (`includeQuery`). The option name still bugs me a bit, because it technically applies to more than just queries, but Apollo uses "query" and "GraphQL document" interchangeably in a lot of places so it's probably fine to match.

Reply:
Whatever we name it, it should also be documented in the readme like `idFromOperation` is.

Reply:
Renamed to `alwaysIncludeQuery` (I think that's what you were suggesting, @lemonmade?). Originally it was "send" to align verbiage wise with the Koa middleware which has a `SendAlways` option.

@gmcgibbon, documented it in the readme.